### PR TITLE
EZS-695 [Studio Demo] - Include trending articles

### PR DESCRIPTION
**Pull request by kamilmusial**
Added recommendations bundle dependencies.
Created template for displaying recommendations in app folder (proper way by EzSystemsRecommendationsBundle)

JIRA: https://jira.ez.no/browse/EZS-695

### CODE REVIEW

**Comment by kamilmusial** [comment]
ping @lserwatka @clash82

File: app/Resources/EzSystemsRecommendationBundle/views/recommendations.html.twig [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<div class="container">
+    <script id="recommendations-template-{{ templateId }}" type="text/x-handlebars-template">
+        {% raw %}
+        <div class="row recommendations">
+            {{#each this}}
+            <a href="{{attr.url}}" onclick="return eZ.RecommendationRestClient.ping('{{clickRecommended}}')">
+                <article class="class-article col-sm-4 col-xs-12">
+                    <div class="recommendations-article">
+                        <div class="col-sm-4 recommendations-thumbnail" style="background-image: url({{attr.image}});">
```
Comment (clash82) [inline app/Resources/EzSystemsRecommendationBundle/views/recommendations.html.twig]: `background: url({{attr.image}});` should do the trick I think :)

File: app/Resources/EzSystemsRecommendationBundle/views/recommendations.html.twig [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                        </div>
+                        <div class="col-sm-8 recommendations-data">
+                            <h3>{{attr.title}}</h3>
+                        </div>
+                    </div>
+                </article>
+            </a>
+            {{else}}
+            <div class="recommendations-message">{% endraw %}{{ 'No recommendations found'|trans }}{% raw %}</div>
```
Comment (clash82) [inline app/Resources/EzSystemsRecommendationBundle/views/recommendations.html.twig]: we could improve indents here

File: app/Resources/EzSystemsRecommendationBundle/views/recommendations.html.twig (hunk 1 above)
Comment (kamilmusial) [inline app/Resources/EzSystemsRecommendationBundle/views/recommendations.html.twig]: We spoke about it, stays like it is :)

**Comment by clash82** [comment]
+1

**Pull request merged**